doc/rados/operations/erasure-code-clay.rst

```diff
@@ -1,8 +1,8 @@
-=========================
-Coupled LAYer code plugin
-=========================
+================
```

Comment:
i'd suggest you to move the doc changes in 864ccaee3231037dec2c6963949f3ad70a2ff719  into this commit . 

you could use `git reset HEAD^^` and `git add -p` to cherry-pick the change to add, for preparing the commits.